.travis.yml

```diff
@@ -29,18 +29,14 @@ version: ~> 1.0
 
 dist: xenial
 os: linux
-language: minimal
+language: shell
```

Comment:
From what I can tell, these are equivalent, why change it?

Reply:
To explicitly define what we are going to use:
![Screenshot from 2020-04-09 02-22-09](https://user-images.githubusercontent.com/32963518/78842630-2c046000-7a09-11ea-91ed-fb70a7d0423a.png)

Reply:
https://docs.travis-ci.com/user/languages/minimal-and-generic/#aliases

Reply:
Reverted.